nestkernel/connection_manager.h

```diff
@@ -393,6 +400,17 @@ ConnectionManager::get_max_delay() const
   return max_delay_;
 }
 
+inline void
+ConnectionManager::append_move_connections_( std::deque< ConnectionID >& input,
```

Comment:
Inlining makes sense here, but the code should go into the `cpp`-file, since the method is used only there. In that way, changes to the code in the method won't require recompilation of all files including `connection_manager.h`.